src/Symbolics.jl

```diff
@@ -197,6 +197,8 @@ for sType in [Pair, Vector, Dict]
     @eval substitute(expr::Arr, s::$sType; kw...) = wrap(substituter(s)(unwrap(expr); kw...))
 end
 
+include("solver/RootFinding.jl")
```

Comment:
follow the naming standards.

Reply:
May you elaborate? thank you

Reply:
See the SciMLStyle guide for details. File names are generally snake case